test/e2e/kubevirt.go

```diff
@@ -1064,15 +1068,39 @@ if [ "$ipv6" != "" ]; then
 	iperf3 -s -D --bind $ipv6 --logfile /tmp/test_${ipv6}_iperf3.log
 fi
 `
-		createIperfServerPods = func(nodes []corev1.Node, netConfig networkAttachmentConfig) ([]*corev1.Pod, error) {
-			annotations := map[string]string{}
-			if netConfig.role != "primary" {
-				annotations["k8s.v1.cni.cncf.io/networks"] = fmt.Sprintf(`[{"name": %q}]`, netConfig.name)
+		nextIPs = func(idx int, subnets []string) ([]string, error) {
+			var ips []string
+			for _, subnet := range subnets {
+				ip, ipNet, err := net.ParseCIDR(subnet)
+				if err != nil {
+					return nil, err
+				}
+				for _ = range idx {
+					ip = iputils.NextIP(ip)
+				}
+				ipNet.IP = ip
+				ips = append(ips, ipNet.String())
 			}
+			return ips, nil
+		}
+		createIperfServerPods = func(nodes []corev1.Node, netConfig networkAttachmentConfig, staticSubnets []string) ([]*corev1.Pod, error) {
 			var pods []*corev1.Pod
-			for _, node := range nodes {
+			for i, node := range nodes {
+				var nse *nadapi.NetworkSelectionElement
+				if netConfig.role != "primary" {
+					staticIPs, err := nextIPs(i, staticSubnets)
```

Comment:
could you remind me why are we creating one server pod for node ?

Reply:
To propertly test east/west at locally and remote node

Reply:
oh right, we're filtering out the control plane node, right ? I was finding it weird that we were creating 3 server pods ... We'd only have two, right ? (assuming kind's default deployment).

Reply:
we just use the workers 
```golang
workerNodeList, err := fr.ClientSet.CoreV1().Nodes().List(context.Background(), metav1.ListOptions{LabelSelector: labels.FormatLabels(map[string]string{"node-role.kubernetes.io/worker": ""})})
```

Reply:
IIUC the aim is to verify connectivity from all nodes